src/visualizr/anitalker/model/nn.py

```
@@ -32,25 +32,27 @@ def conv_nd(dims, *args, **kwargs):
-    elif dims == 2:
-        return nn.AvgPool2d(*args, **kwargs)
-    elif dims == 3:
-        return nn.AvgPool3d(*args, **kwargs)
+    match dims:
+        case 1:
+            return nn.AvgPool1d(*args, **kwargs)
+        case 2:
+            return nn.AvgPool2d(*args, **kwargs)
+        case 3:
+            return nn.AvgPool3d(*args, **kwargs)
     raise ValueError(f"unsupported dimensions: {dims}")
```

Ruff (0.12.2)

Avoid specifying long messages outside the exception class

(TRY003)